docs/application/native/guides/internationalization/i18n.md

```diff
@@ -2646,6 +2656,9 @@ To format measure objects:
    i18n_measure_format_h format;
    i18n_measure_format_create(language, country, I18N_UMEASFMT_WIDTH_WIDE, &measure_format);
    ```
+   > **Note**  
+   > This function is deprecated from Tizen 5.0. Instead. you can use `i18n_measure_format_create_from_locale_id()`,
+   > which takes the locale ID parameter. To set the language as en and country as US, use the locale ID en_US.
```

Comment:
Instead, use `i18n_measure_format_create_from_locale_id()`, which takes the locale ID parameter. To set the language as "en" and country as "US", use the locale ID "en_US".